components/FormFieldSelect/FormFieldSelect.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from 'react';
+import { Select } from '../Select';
+
+interface FormFieldSelectItem {
+  value: string;
+  label: string;
+}
+
+interface FormFieldSelectProps {
+  label: string;
+  options: FormFieldSelectItem[];
+  themeColor: 'light' | 'dark';
```

Comment:
Just name it `theme`, and make it option. Default to `light`.

Reply:
Updated